### ISSUE TITLE
plan move to Slack Enterprise Grid

### ISSUE THREAD

**Issue by afeld** [18F/tts-tech-portfolio#501]
## Background information

GSA IT is going to require that all Slack workspaces at GSA be under the same Enterprise Grid instance to simplify management, e-discovery, etc. We will be hopping on their contract.

## User stories

- As someone administering Slack workspaces, I want to do so in one place so that the configuration is consistent.
- As someone responsible for e-discovery of Slack content, I want it to integrate with my tools for doing so.

## Acceptance criteria

- [x] Finance is aware of what it will cost for the next 1-1.5 contract cycles - [Slack message](https://gsa-tts.slack.com/archives/G148EPNQJ/p1592427105001500?thread_ts=1592426787.001400&cid=G148EPNQJ)
- [ ] Finance agrees to it
- [ ] Funds identified
- [x] There is a (tentative) cutover date agreed upon by GSA IT and TTS

---

The assignee should add some checkboxes as a "sketch" of the steps to complete, which may evolve.

**Comment by afeld** [comment]
- [IGCE](https://docs.google.com/spreadsheets/d/1glNehl9aonJUJ2Z2Vb3B1fnV6rd1Ti7RlyizFkegjTE/edit?ts=5ed91899#gid=0)
- [Before and after cost estimation](https://docs.google.com/spreadsheets/d/1KH7A7NQ5bvAqzJ63rOt403FNvfxczOoB4e2F2CDk6TA/edit#gid=497138473)
- [Q&A doc](https://docs.google.com/document/d/1smXVRPIbfFCBczRtWJ0a-6KWM2yaW-wXMHie-vn7EUQ/edit?ts=5ed016af#)

Current status: waiting to hear back from Slack on the various [cost scenarios](https://docs.google.com/document/d/1smXVRPIbfFCBczRtWJ0a-6KWM2yaW-wXMHie-vn7EUQ/edit?ts=5ed016af#bookmark=id.xu49i7fl3665).

**Comment by afeld** [comment]
Notified Finance with an estimate; waiting on Slack to get a better estimate of how many licenses we will need.

**Comment by afeld** [comment]
Response re: user count:

> thus far your peak in terms of paid users was 430 and has been steadily in the 400's to 430 over the past few months

**Comment by its-a-lisa-at-work** [comment]
@afeld can you say what is blocked about this card in a comment for reference purposes?

**Comment by afeld** [comment]
Identifying funds to pay for it.

**Comment by afeld** [comment]
Closing in favor of https://github.com/18F/tts-tech-portfolio/issues/678.
